src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  background: $color-white;
+  border: 1px solid $color-second-white;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  &__button-container {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: $color-blue;
+    border: 0;
+    margin-top: 16px;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    margin-top: 4px;
+    font-family: $font-family;
+    font-weight: $font-width;
+    font-size: $font-size-first;
+    color: #616070;
+  }
+
+  &__price-label {
+    align-items: flex-end;
+    width: 31px;
+    line-height: 18px;
+    font-family: $font-family;
+    font-style: normal;
+    font-weight: $font-width;
+    font-size: $font-size-second;
+    color: #616070;
+    margin-right: 85px;
+  }
+
+  &__price {
+    align-items: flex-end;
+    text-align: right;
+    width: 50px;
+    line-height: 18px;
+    font-family: $font-family;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-dark;
```

Comment:
```suggestion
    color: $color-dark;
    
    &-name {},
    &-container {}
```

Reply:
it means you can make  `&__price-container` and `&__price-label` selectors nested into this